idp/styling/main.css

```diff
@@ -1,154 +0,0 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
```

Comment:
Is there a way to incorporate this CSS to the new login page?

Reply:
I could probably figure it out. But is it important for testing?

Reply:
I'd say it is not extremely important, but it is kind of a regression, especially for the `demo` site since the l&f matches between the login page and site. We can talk about it in the PASS status call tomorrow to see what others think?

Reply:
Agreed in slack pass-dev channel to leave as is for now.